tests/src/Kernel/Entity/EntityGroupsTest.php

```diff
@@ -0,0 +1,216 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\Tests\og\Kernel\Entity\EntityGroupsTest.
+ */
+
+namespace Drupal\Tests\og\Kernel\Entity;
+
+use Drupal\entity_test\Entity\EntityTest;
+use Drupal\KernelTests\KernelTestBase;
+use Drupal\Component\Utility\Unicode;
+use Drupal\og\Entity\OgMembership;
+use Drupal\og\Og;
+use Drupal\user\Entity\User;
+
+/**
+ * Tests entity group methods
+ *
+ * @group og
+ */
+class EntityGroupsTest extends KernelTestBase {
```

Comment:
the way I see it isMember is just sugar on top of  `getEntityGroups`. If you don't agree, it's minor we can keep as-is.